sentry.config.js

```diff
@@ -25,6 +25,24 @@ export const sentryOptions = {
     // Common non-actionable errors
     "rejected transaction",
     "User closed modal",
+    "Loading chunk",
+    "Failed to execute '",
+    "NetworkError when attempting to fetch resource.",
+    "googlefc is not defined",
+    "__cmp is not defined",
+    "Cannot read properties of undefined (reading 'cmp')",
+    "Cannot read properties of undefined (reading 'loaded')",
+    "Cannot read properties of undefined (reading 'outputCurrentConfiguration')",
+    "apstagLOADED is not defined",
+    "moat_px is not defined",
+    "ResizeObserver loop limit exceeded",
+    "null is not an object (evaluating 'e.uid')",
+    "window.ReactNativeWebView.postMessage is not a function",
+    "_reportEvent is not defined",
+    "requestAnimationFrame is not defined",
+    "window.requestAnimationFrame is not a function",
```

Comment:
technically these mean the user is on an outdated browser, not *not* actionable but I agree they can be ignored for now since we are not gonna go support random year old browsers atm